source/shared/cpp/ObjectModel/Container.cpp

```diff
@@ -56,6 +57,28 @@ void Container::SetSelectAction(const std::shared_ptr<BaseActionElement> action)
     m_selectAction = action;
 }
 
+void Container::SetLanguage(const std::string& value)
+{
+    for (auto& item : m_items)
+    {
+        CardElementType elementType = item->GetElementType();
+
+        if (elementType == CardElementType::Container)
```

Comment:
>Container [](start = 44, length = 9)

Do we also need to handle ColumnSet?

Reply:
aside from Container, Show Card Actions and Column Sets, are there other places where a TextBlock could be inserted?

---
In reply to: [170748153](https://github.com/Microsoft/AdaptiveCards/pull/1102#discussion_r170748153) [](ancestors = 170748153)

Reply:
I think that's it

---
In reply to: [170760621](https://github.com/Microsoft/AdaptiveCards/pull/1102#discussion_r170760621) [](ancestors = 170760621,170748153)